cob_default_robot_config/robots/cob4-9/upload_param_cob4-9.launch

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<launch>
+
+	<!-- send ROBOT parameters to parameter server -->
+	<rosparam command="load" ns="/script_server/base" file="$(find cob_default_robot_config)/robots/cob4-9/base_configurations.yaml"/>
```

Comment:
please obey new structure, i.e. adjust to yaml files being in `script_server` subfolder